### ISSUE TITLE
Bottle framework logo license permissions

### ISSUE THREAD

**Issue by ibakirov** [bottlepy/bottle#1400]
I have drafted Wikipedia page for the [Bottle framework](https://en.wikipedia.org/wiki/Draft:Bottle_(web_framework)) and it is being reviewed by Wikipedia staff. Well the question is about the Bottle's logo license.

@defnull Do you permit to upload current logo picture to Wikipedia page? Or maybe release current logo for permissive license to use for referral to the Bottle framework Website/GitHub Project page?

**Comment by defnull** [comment]
Thanks for your engagement! There was an article once, but it was removed at some point for reasons I do not remember. It was too short I guess?

As for the logo image, it is not licensed as Open Source or CC at the moment, but I tried to cover this exact scenario in `README.rst`:

> It is allowed to use the logo as a link to the bottle homepage or in direct context with the unmodified library.

So for an article covering Bottle (and not a fork), usage is explicitly allowed. Not sure if that is enough for Wikimedia standards, but the conditions are similar in nature to the ones attached to the Flasks logo. I can release the original SVG version under the same conditions if you want.
